frontend/.eslintrc.js

```diff
@@ -86,6 +86,7 @@ module.exports = {
         schemaString: fs.readFileSync('../pkg/graphql/schema.graphql', 'utf-8'),
       },
     ],
+    'tsdoc/syntax': 'warn',
```

Comment:
We should aim to make this an error instead of a warning. Are there problems that prevent us from using error currently?